### PR TITLE
Rozetka test#13

### PR DESCRIPTION
**Pull request by grinred**

### CODE REVIEW

File: src/test/java/RozetkaSergey/RozetkaPages/RozetkaPages5_34.java [hunk 1]
```diff
@@ -64,7 +65,26 @@ public WebElement affirmDelete() {
...
+    public List<WebElement> listOfTelefonNumbers (){return getElements("//section[@class='contacts-section']//div[@class='contacts-main__cell']//a[@class='contacts-main__number']");}
 
+
+
+    public void checkingListOfCities(){
+        List<WebElement> cities = listOfCites();
+        for (int i = 0; i<5; i++){
+            System.out.println(cities.get(i).getText());
+        }
+
```
Comment (Znamm9) [inline src/test/java/RozetkaSergey/RozetkaPages/RozetkaPages5_34.java:84]: its a shame to have so many empty lines!)

File: src/test/java/RozetkaSergey/TestsRozetka/CheckContact.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        closeForeverAd();
+        rozetkaPages5_34.contactButton().click();
+        isElementPresent();
+        closeForeverAd();
+        String tel = rozetkaPages5_34.listOfTelefonNumbers().get(0).getText();
+        Assert.assertEquals(stringToComp, tel);
+    }
+
+    public boolean isElementPresent(){
+        try {
```
Comment (Znamm9) [inline src/test/java/RozetkaSergey/TestsRozetka/CheckContact.java:26]: you can change those lines to simplier format
```suggestion
        try {
            return rozetkaPages5_34.closeAdvertisement().isEnabled();
            } catch (Exception e) {
            return false;
        }
```

File: src/test/java/RozetkaSergey/TestsRozetka/CheckContact.java [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+            rozetkaPages5_34.closeAdvertisement().isEnabled();
+            return myBooleon = true;
+        } catch (Exception e) {
+            return myBooleon = false;
```
Comment (Znamm9) [inline src/test/java/RozetkaSergey/TestsRozetka/CheckContact.java:30]: Please put more informative variables names for example:
isElementEnabled

File: src/test/java/RozetkaSergey/TestsRozetka/CheckContact.java [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+        }
+    }
+    public void closeForeverAd() {
+        if (myBooleon) {
+
```
Comment (Znamm9) [inline src/test/java/RozetkaSergey/TestsRozetka/CheckContact.java:35]: what is this ***? empty method without any data. remove it please

File: src/test/java/RozetkaSergey/TestsRozetka/SwichLaunguage.java [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+import ClasesToAllUs.TestInit;
+import RozetkaSergey.RozetkaPages.RozetkaPages5_34;
+import org.openqa.selenium.By;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class SwichLaunguage extends TestInit {
+    String stringToComp = "Знайти";
+    RozetkaPages5_34 rozetkaPages5_34;
+    boolean myBooleon;
```
Comment (Znamm9) [inline src/test/java/RozetkaSergey/TestsRozetka/SwichLaunguage.java:12]: Add empty line before method annotation

File: src/test/java/RozetkaSergey/TestsRozetka/SwichLaunguage.java [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+        rozetkaPages5_34.clickOnThreeStrips().click();
+        rozetkaPages5_34.changLanguageButton().click();
+        String textFromButton = rozetkaPages5_34.fiendButton().getText();
+        Assert.assertEquals(textFromButton,stringToComp);
+        sleep(1);
+        isElementPresent();
+        closeForeverAd();
+    }
+    public boolean isElementPresent(){
+        try {
```
Comment (Znamm9) [inline src/test/java/RozetkaSergey/TestsRozetka/SwichLaunguage.java:28]: your code is not DRY! Above you have totally the same method. Try to reuse your code

File: src/test/java/RozetkaSergey/TestsRozetka/SwichLaunguage.java [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+            rozetkaPages5_34.closeAdvertisement().isEnabled();
+            return myBooleon = true;
+        } catch (Exception e) {
+            return myBooleon = false;
+        }
+    }
+    public void closeForeverAd() {
+        if (myBooleon) {
```
Comment (Znamm9) [inline src/test/java/RozetkaSergey/TestsRozetka/SwichLaunguage.java:36]: it seems you copied 2 java files.... please do your job properly

File: src/test/java/RozetkaSergey/TestsRozetka/SwitchCity.java [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+        rozetkaPages5_34.enterToRozetka("https://rozetka.com.ua/");
+        rozetkaPages5_34.closeAdvertisement().click();
+        rozetkaPages5_34.clickOnThreeStrips().click();
+        rozetkaPages5_34.chooseCity().click();
+        String one = rozetkaPages5_34.listOfCites().get(3).getText();
+        rozetkaPages5_34.listOfCites().get(3).click();
+        rozetkaPages5_34.klickAsept().click();
+        System.out.println(one+" one City of the list");
+        Assert.assertEquals(one,"Запорожье");
+        sleep(1);
```
Comment (Znamm9) [inline src/test/java/RozetkaSergey/TestsRozetka/SwitchCity.java:21]: no need to do sleep after assertion

Comment (grinred) [reply src/test/java/RozetkaSergey/TestsRozetka/SwitchCity.java:21]: i done

File: src/test/java/RozetkaSergey/RozetkaPages/RozetkaPages5_34.java (hunk 1 above)
Comment (grinred) [reply src/test/java/RozetkaSergey/RozetkaPages/RozetkaPages5_34.java:84]: done

File: src/test/java/RozetkaSergey/TestsRozetka/CheckContact.java (hunk 2 above)
Comment (grinred) [reply src/test/java/RozetkaSergey/TestsRozetka/CheckContact.java:26]: done

File: src/test/java/RozetkaSergey/TestsRozetka/CheckContact.java (hunk 3 above)
Comment (grinred) [reply src/test/java/RozetkaSergey/TestsRozetka/CheckContact.java:30]: done

File: src/test/java/RozetkaSergey/TestsRozetka/CheckContact.java (hunk 4 above)
Comment (grinred) [reply src/test/java/RozetkaSergey/TestsRozetka/CheckContact.java:35]: done

File: src/test/java/RozetkaSergey/TestsRozetka/SwichLaunguage.java (hunk 5 above)
Comment (grinred) [reply src/test/java/RozetkaSergey/TestsRozetka/SwichLaunguage.java:12]: done

**Review by StepaniukAndii — APPROVED** [review]

**Pull request merged**